packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.test.js

```diff
@@ -101,3 +101,9 @@ test('renders secondary with event handlers', async () => {
 
     expect(onRemoveItemClick).toHaveBeenCalledTimes(1)
 })
+
+test('renders the Remove button', () => {
```

Comment:
Why do we need this new test here?
We have a test to check the component rendering above. `renders secondary action component`